libraries/lib-music-information-retrieval/tests/TatumQuantizationFitVisualization/visualize_debug_output.py

```diff
@@ -0,0 +1,39 @@
+from debug_output import *
+
+# Make a plot of the ODF, with peaks marked. Add another layer where the tatums are shown, for example as vertical lines.
+import matplotlib.pyplot as plt
+import math
+
+plt.figure(1)
+# Plot the ODF, using odfSr to convert from samples to seconds
+plt.plot([i/odfSr for i in range(len(odf))], odf)
+# Plot the peaks
+plt.plot([i/odfSr for i in odf_peak_indices], [odf[i] for i in odf_peak_indices], 'ro')
+# Convert tatum rate from tatums per minute to tatums per ODF sample:
+tatumRate = tatumRate / 60.0
+
+# Plot the tatums as vertical lines, offset by `lag` samples
+numTatums = int(math.ceil(len(odf) / odfSr * tatumRate))
```

Comment:
But here, if the comment is right, `tatumRate` is tatums / second.  So the previous comment is what's wrong.

Reply:
Addressed.